add-license-text

```diff
@@ -1,5 +1,5 @@
 
-Copyright 2020 Actyx AG
+Copyright 2021 Actyx AG
```

Comment:
Is there a specific reason for choosing the Apache license? Asking b/c I'd typically default to MIT. Saw Academy-Data is also under Apache.

Reply:
We had this conclusion actyx internal after setting the Pond lic.